src/aktualizr_lite/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+if(BUILD_OSTREE)
+add_executable(aktualizr-lite main.cc)
+target_link_libraries(aktualizr-lite aktualizr_static_lib ${AKTUALIZR_EXTERNAL_LIBS})
+
+install(TARGETS aktualizr-lite RUNTIME DESTINATION bin COMPONENT aktualizr-lite)
+
+set(TEST_SOURCES test_list.sh)
+
+if((CMAKE_BUILD_TYPE MATCHES "Valgrind") AND (NOT AKTUALIZR_TEST_NO_VALGRIND))
+    add_custom_target(t_aktualizr-lite-list
+        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/test_list.sh
+        $<TARGET_FILE:aktualizr-repo> ${PROJECT_SOURCE_DIR} ${RUN_VALGRIND})
+else()
+    add_custom_target(t_aktualizr-lite-list
+        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/test_list.sh
+        $<TARGET_FILE:aktualizr-repo> ${PROJECT_SOURCE_DIR})
+endif()
+
+add_dependencies(t_aktualizr-lite-list aktualizr-repo)
+add_dependencies(t_aktualizr-lite-list aktualizr-lite)
+add_dependencies(build_tests t_aktualizr-lite-list)
+
+add_test(NAME t_aktualizr-lite-list
```

Comment:
Did you really intend to keep the `t_aktualizr-lite-list` custom target alongside the `t_aktualizr-lite-list` test run by ctest? The fact that they share the same name is a bit confusing and they run the same command with diferrent arguments.

I would expect that you'd only need two `add_test`, one with and without valgrind?